CONTRIBUTING.md

```diff
@@ -61,8 +61,8 @@ Create a new directory in the `packages` directory with the name of the loader
   "name": "@nodejs-loaders/YOUR-LOADER",
   "description": "Extend node to support YOUR THING via customisation hooks.",
   "type": "module",
-  "main": "./YOUR-LOADER.mjs",
-  "types": "./YOUR-LOADER.d.mts",
+  "main": "./YOUR-LOADER.js",
+  "types": "./YOUR-LOADER.d.ts",
   "author": "YOUR NAME",
```

Comment:
Why? The `.mjs` is important.

Also, this is a breaking change.

Reply:
oh shit JSR only accept .js.
If need maybe open issue on JSR